lib/phases.py

```diff
@@ -417,6 +417,7 @@ class SwiftExecutable(BuildPhase):
     def __init__(self, executableName, sources):
         BuildAction.__init__(self, output=executableName)
         self.executableName = executableName
+        self.name = executableName
```

Comment:
Adding a dependency of the executable creation for XDG  to foundation_tests (which is the SwiftExecutable: Test Foundation) in build.py caused a script failure here as the 'name' seemed to be 'nil'. Hence, the change.